src/app/components/header/header.component.css

```diff
@@ -0,0 +1,30 @@
+nav{
+    letter-spacing: 2.5px;
+    margin-top: 7px;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.right-nav{
+    display: flex;
+}
+
+.left-nav li{
+    display: inline;
+    margin-right: 50px;
+}
+
+button{
+    font-size: small;
+}
+
+a{
```

Comment:
bad practice to define styles using element selectors - use classes instead - as this will cause problems when you need to add additional buttons or anchor links to your header

Reply:
Okay, understood!